apps/mobile/src/features/send/send-container.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import { Account } from '@/store/accounts/accounts';
+import { HasChildren } from '@/utils/types';
+
+import { CryptoCurrency } from '@leather.io/models';
+
+import { SendContext, SendProvider, SendStep } from './send-context';
+
+export function SendContainer({ children }: HasChildren) {
```

Comment:
Maybe we can consider using `useReducer` here instead of `useState` ? 

When there are a few state things to keep track of sometimes its a nicer way of managing things. We can use `dispatch` and `actions` then to update an object then rather than a few separate state items

Reply:
Good point, even if I don't refactor it here, I will as I start to expand on this approach.